apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html

```diff
@@ -2,19 +2,42 @@
     <div class="main-toolbar-left">
         <div class="content-type__title">
             <header>
-                <h4>{{ contentType.name }}</h4>
+                <dot-icon name="{{ contentType.icon }}"></dot-icon>
+                <p-inplace #contentTypeInlineEdit (onActivate)="editInlineActivate($event)">
+                    <ng-template pTemplate="display">
+                        <h4>{{ contentType.name }}</h4>
+                    </ng-template>
+                    <ng-template pTemplate="content">
+                        <input
+                            [style.width.px]="contentTypeNameInputSize"
+                            #contentTypeNameInput
+                            type="text"
+                            [value]="contentType.name"
+                            (keydown.enter)="fireChangeName()"
+                            (keydown.escape)="contentTypeInlineEdit.deactivate()"
+                            pInputText
+                            dotAutofocus
+                        />
+                    </ng-template>
+                </p-inplace>
```

Comment:
Let's move this to a component so that we can reuse it in edit mode.

Make sure that we can pass the content as children so we can have any HTML inside.